src/app/wallet/view.nim

```diff
@@ -457,11 +457,16 @@ QtObject:
   proc gasPricePredictionsChanged*(self: WalletView) {.signal.}
 
   proc getGasPricePredictions*(self: WalletView) {.slot.} =
-    let prediction = self.status.wallet.getGasPricePredictions()
-    self.safeLowGasPrice = $prediction.safeLow
-    self.standardGasPrice = $prediction.standard
-    self.fastGasPrice = $prediction.fast
-    self.fastestGasPrice = $prediction.fastest
+    spawnAndSend(self, "getGasPricePredictionsResult") do:
+      let walletModel = newWalletModel(nil)
```

Comment:
I didn't know you could do that. It's good to know.
The way I did it is not as clean, but uses less memory:
```nim
let walletmodel = self.status.wallet
spawnAndSend(self, "getGasPricePredictionsResult") do:
       $ %walletModel.getGasPricePredictions()
```

Reply:
Nice, updated, thanks!